src/components/custom-field-input-single-choice/__snapshots__/custom-field-input-single-choice.test.jsx.snap

```diff
@@ -40,7 +40,7 @@ exports[`src/components/custom-field-input-single-choice/custom-field-input-sing
               id="test-id"
               name="field-id"
               role="combobox"
-              style="--numIcon: 1;"
+              style="--numIcon: 3;"
```

Comment:
Instead of a dynamic number, it is hard-coded to 3 -- that is the max number of icons on the select component. We don't want to allow test to underflow into this padded area because a feature team might end up sizing the component too small to fit when all 3 icons are shown.